README.md

```diff
@@ -1,2 +1,136 @@
+
+**[Ronald_Mjonono_Portfolio]** is a sneak peak into my abilities as a software developer
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+Built with HTML and CSS
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/](https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Showcase of projects and personal skills I have as a software developer
+- Projects displayed are available live on provided links
+
+- **[Personal_Projects]**
+- **[Design_SKills]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Link to be added
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Open terminal on your computer, clone the project, go live and view the portfolio
+
+### Prerequisites
+
+In order to run this project you need:
+
+A computer with terminal
+
+### Setup
+
+Clone this repository to your desired folder:
+git clone https://github.com/mj-e-boc/M-J-Portfolio.git
+
+open the project using:
+cd folder-name
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@mj-e-boc](https://github.com/mj-e-boc)
+- Twitter: [@mjononoronald](https://twitter.com/Mjononoronald)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ronald-mjonono-86365988/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Recent_projects]**
+- [ ] **[Contact_section]**
+- [ ] **[Desktop_view]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/mj-e-boc/M-J-Portfolio/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to get in touch with me via a message here or contact platforms listed above.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the microverse Alumni that keep inspiring me to code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[How_to_get_in_touch]**
+
+  - [check_contact_section_above]
+
+- **[You_open_for_new_ideas]**
+
+  - [The_more_the_better]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
I can see you created a license file as requested by the previous reviewer👍. But currently, you still haven't updated the license link🥲. Kindly update the license link with the link to the created license file in your repo😉. When this is implemented, it should lead to your license file😉.


```suggestion
This project is [MIT](./LICENSE) licensed.
```

Try the suggestion above if it works, but I think that should work😉.